packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import "@rari-capital/solmate/src/tokens/ERC721.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "./SocialContract.sol";
+
+contract Optimist is ERC721 {
```

Comment:
The ERC721 name and symbol are set in storage in the constructor so they won't be set in the proxy, we need to set them in an initialize using assembly

Reply:
We will need to either have an assertion that the string is less than 32 bytes or implement the solidity logic at a low level for storing strings larger than 32 bytes

Reply:
Perhaps using oz contracts-upgradable is more simple here